packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -22,23 +86,76 @@ const TeamPromptMeeting = (props: Props) => {
     graphql`
       fragment TeamPromptMeeting_meeting on TeamPromptMeeting {
         ...TeamPromptTopBar_meeting
+        ...useMeeting_meeting
+        phases {
+          stages {
```

Comment:
-1 fragment on `TeamPromptResponsesPhase` instead. it's a little more expressive way to say "we only care about this 1 phase in particular"

as it's written, it's saying, "look at EVERY phase & for EVERY phase, give me the TeamPromptResponsesStages"

Also, if you fragment on the phase, then the teamMember object is guaranteed non null, which is nice.

Reply:
oh cool, will fix